src/formatDate.js

```diff
@@ -1,25 +1,41 @@
 'use strict';
 
-/**
- * Time flies, standards change. Let's get rid of the routine of changing the date format,
- * and create a function for formatting dates.
- * Create a `formatDate` function that accepts the `date` string, the old `fromFormat` array variable,
- * and the new `toFormat` array variable. Function returns given date in `toFormat` format.
- * 
- * Example:
- * formatDate('2020-02-18', ['YYYY', 'MM', 'DD', '-'], ['DD', 'MM', 'YY', '/']) // '18/02/20'
- * formatDate('2021-02-18', ['YYYY', 'MM', 'DD', '-'], ['DD', 'MM', 'YY', '/']) // '18/02/21'
- * formatDate('97/02/18', ['YY', 'MM', 'DD', '/'], ['DD', 'MM', 'YYYY', '.']) // '18.02.1997'
- *
- * @param {string} date
- * @param {string[]} fromFormat
- * @param {string[]} toFormat
- *
- * @returns {string}
- */
-
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const values = date.split(fromFormat[3]);
+  const result = new Array(3);
+
+  for (const [index, value] of toFormat.entries()) {
+    switch (value) {
+      case 'DD':
+        result[index] = values[fromFormat.indexOf(value)];
+        break;
+      case 'MM':
+        result[index] = values[fromFormat.indexOf(value)];
+        break;
+      case 'YYYY':
+        if (fromFormat.includes(value)) {
+          result[index] = values[fromFormat.indexOf(value)];
+        } else if (fromFormat.includes('YY')) {
+          if (values[fromFormat.indexOf('YY')] > 20) {
+            result[index] = 19 + values[fromFormat.indexOf('YY')];
+          } else {
+            result[index] = 20 + values[fromFormat.indexOf('YY')];
+          }
+        }
+        break;
+      case 'YY':
+        if (fromFormat.includes(value)) {
+          result[index] = values[fromFormat.indexOf(value)];
+        } else if (fromFormat.includes('YYYY')) {
+          result[index] = values[fromFormat.indexOf('YYYY')].substring(2);
+        }
+        break;
+      default:
+        break;
```

Comment:
maybe it's better to write 'Invalid values' before the break to show that values didn't match those you expected